ceph_deploy/hosts/fedora/__init__.py

```diff
@@ -18,7 +18,7 @@ def choose_init():
 
     Returns the name of a init system (upstart, sysvinit ...).
     """
-    return 'sysvinit'
+    return 'systemd'
```

Comment:
Would you be open to at least making it a clear cut-over with F22, like it's done with Yum-> DNF?

https://github.com/ceph/ceph-deploy/blob/master/ceph_deploy/hosts/fedora/__init__.py#L25-L28

But yeah, I see your point about versions.  That will take a bit more thought.

Are you picturing wanting to release systemd-enabled packages on F22?  Unless such packages are going to show up, it's not an issue for Hammer. 

But for Infernalis (maybe?) and definitely Jewel, we will have packages that are now systemd where we were sysvinit before (-EL7 anything).  So yeah, we'll have to be smart enough for that.   :/

This is a problem that has to be solved anyways, because we will also need to make different choice when it comes to osd/mon package split depending on Ceph version.

I think a simple table can be used for named releases. but for dev/gitbuilder builds, that's a bit uglier.  Then we ahve to start hardcoding entries of "if version >= 9.x....".  We'll see.